### PR TITLE
Fix reader access to job log

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: scenarios/utils/job/job_utils.go [hunk 1]
```diff
@@ -10,8 +10,38 @@ import (
...
+	jobSummaries, err := getPipelineJobs(cfg, appName)
+	if err != nil {
+		return nil, err
+	}
+	lastJobSummary := jobSummaries[0]
+	if lastJobSummary.Status != expectedStatus {
+		return nil, fmt.Errorf("method GetLastPipelineJobWithStatus for application %s expected status '%s', but it received '%s'",
+			appName, expectedStatus, lastJobSummary.Status)
+	}
+	logger.Debugf("method GetLastPipelineJobWithStatus for application %s received expected status '%s'", appName, expectedStatus)
```
Comment (Pespiri) [inline scenarios/utils/job/job_utils.go:24]: is it necessary to log to debug console?

File: scenarios/utils/job/job_utils.go [hunk 2]
```diff
@@ -10,8 +10,38 @@ import (
...
+
+// GetAnyPipelineJobWithStatus Checks if any pipeline job exists with status
+func GetAnyPipelineJobWithStatus(cfg config.Config, appName, expectedStatus string, logger *log.Entry) (*models.JobSummary, error) {
+	jobSummaries, err := getPipelineJobs(cfg, appName)
+	if err != nil {
+		return nil, err
+	}
+	for _, jobSummary := range jobSummaries {
+		if jobSummary.Status == expectedStatus {
+			logger.Debugf("method GetAnyPipelineJobWithStatus for application %s received expected status '%s'", appName, expectedStatus)
```
Comment (Pespiri) [inline scenarios/utils/job/job_utils.go:36]: is it necessary to log to debug console?

**Review by Pespiri — APPROVED** [review]

File: scenarios/utils/job/job_utils.go (hunk 1 above)
Comment (satr) [reply scenarios/utils/job/job_utils.go:24]: Good to have this option in prod

File: scenarios/utils/job/job_utils.go (hunk 2 above)
Comment (satr) [reply scenarios/utils/job/job_utils.go:36]: Good to have this option in prod

**Pull request merged**